### PR TITLE
Improve doc of  managed resources

### PR DESCRIPTION
**Pull request by Motouom**
**What this PR does / why we need it**: Improve documentation of managed resources

**Which issue this PR fixes** #826 

**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [x] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: docs/MANAGED.md [hunk 1]
```diff
@@ -1,56 +1,68 @@
...
-| Message Bundles                 | Only message bundles imported with config-cli will be managed/deleted.           | `message-bundles`                |
-
-## Disable deletion of managed entities
-
-If you don't delete properties of a specific type, you can disable this behavior by default a properties like `import.managed.<entity>=<full|no-delete>`, e.g.:
-`import.managed.required-actions=no-delete`
-
-## State management
-
-If `import.remote-state.enabled` is set to `true` (default value), keycloak-config-cli will purge only resources they created before by keycloak-config-cli. If `import.remote-state.enabled` is set to `false`, keycloak-config-cli will purge all existing entities if they are not defined in import json.
```
Comment (AssahBismarkabah) [inline docs/MANAGED.md:49]: Hi @Motouom  why remove this section ?

**Review by AssahBismarkabah — CHANGES_REQUESTED** [review]

**Comment by AssahBismarkabah** [comment]
@Motouom  please resolve the conflicts
thank you

**Comment by Motouom** [comment]
> @Motouom please resolve the conflicts thank you

@AssahBismarkabah  thank you, i have solved  the issue

File: docs/MANAGED.md [hunk 2]
```diff
@@ -1,56 +1,72 @@
-# Full managed resources
+# Resource Management in keycloak-config-cli
```
Comment (AssahBismarkabah) [inline docs/MANAGED.md:1]: hi @Motouom  please aligned the tag with ## and ### for subsections, also please just keep the tag 1 as Resource  management

**Review by AssahBismarkabah — APPROVED** [review]

**Pull request merged**